### ISSUE TITLE
[CVE-2021-44832] Update log4j to 2.17.1

### ISSUE THREAD

**Issue by CClerger** [opensagres/xdocreport#521]
Hello, are you planning to release a new version using the latest version of log4j to avoid this vulnerability (https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44832)? If so, when do you think it will be released?

**Comment by angelozerr** [comment]
We have already done a release last week. 

I don't know today when we will do a new release. Any PR are welcome to fix the issue.

**Comment by pascalleclercq** [comment]
@CClerger can you create a PR with a good version of log4j.
Please note that log4j is not a transitive dependency of xdocreport, you are free to use another logging framework such as logback.

**Comment by CClerger** [comment]
The direct dependency to log4j in core is optional, but xdocreport.converter.docx.docx4j depends on version 2.8.1 of docx4j which depends on log4j 1.2.15 (versions 8.0.0 and above of docx4j don't include log4j and let you use your own logging framework). There might be similar dependencies in components I don't use in my project.
